third_party/move/move-compiler-v2/tests/no-simplifier/constant_folding_ristretto.exp

```diff
@@ -1,27 +1,19 @@
 // -- Model dump before bytecode pipeline
 module 0xcafe::Ristretto {
     public fun test() {
-        {
```

Comment:
This exp update does not look correct due to an incorrect setting on  setting of `no-simplifer` in testsuite.rs mentioned below.

Reply:
Good catch, done.